src/domain/youthProfile/edit/__tests__/EditYouthProfile.test.tsx

```diff
@@ -35,7 +35,7 @@ const getWrapper = () => {
   );
 };
 
-test('form has values', async () => {
+test.skip('form has values', async () => {
```

Comment:
A todo explaining why the skip and what needs to be done to activate the test can be helpful